pkgs/applications/misc/archiver/default.nix

```diff
@@ -14,7 +14,7 @@ buildGoModule rec {
     sha256 = "1py186hfy4p69wghqmbsyi1r3xvw1nyl55pz8f97a5qhmwxb3mwp";
   };
 
-  vendorSha256 = "1y4v95z1ga111g3kdv5wvyikwifl25f36firf1i916rxli6f6g5i";
+  vendorHash = "1y4v95z1ga111g3kdv5wvyikwifl25f36firf1i916rxli6f6g5i";
```

Comment:
This won't work.

Reply:
Managed to corrupt my .git, though detected all this cases with:

`git show HEAD | rg vendorHash | rg -v "(null|sha256)"`